### DISCUSSION TITLE
How you can get the Public Sponsor Badge

### DISCUSSION THREAD

**Discussion by caioagiani** [community/community#19916] [Other Feature Feedback, Questions, & Ideas] [answered]
It's pretty simple, you just need to make a donation to some open source contributor.

I will leave my profile below if you want to support me with 1$ (custom value) and in return receive this beautiful Badge

https://github.com/sponsors/caioagiani

![image](https://user-images.githubusercontent.com/56451511/185136355-f81e07d3-9948-49ad-8de8-b4e820aa2fff.png)

**Comment by githubfollow** [comment, marked as answer]
UP!

**Reply by caioagiani** [reply]
Tks!

**Reply by nijige** [reply]
Top

**Comment by fernandopredes** [comment]
Up!

**Reply by caioagiani** [reply]
Tks

**Comment by AftabAhmedAbro** [comment]
Kindly write in detail about it
Thanks

**Reply by caioagiani** [reply]
Very simple, just support an open source contributor to get your "Sponsor" badge

**Comment by gabyreis28** [comment]
up

**Reply by caioagiani** [reply]
> up

Tks

**Reply by gabyreis28** [reply]
ty

**Comment by KingOfChelsea** [comment]
up

**Reply by caioagiani** [reply]
Tks

**Comment by ghost** [comment]
Up

**Reply by caioagiani** [reply]
Tks

**Comment by Nayab007** [comment]
thanks

**Reply by caioagiani** [reply]
:)

**Comment by Nayab007** [comment]
Add my

**Reply by caioagiani** [reply]
Ok!

**Comment by Rods2000** [comment]
Up

**Reply by caioagiani** [reply]
Thank you for your sponsorship and congratulations on your Badge 🥳

**Reply by Rods2000** [reply]
My pleasure...keep doing great work! Thanks for the Badge, how do I claim it or is it automatic?

**Reply by caioagiani** [reply]
Looks like you already got your badge :partying_face:

![image](https://user-images.githubusercontent.com/56451511/194866835-d4a29744-aa95-4e59-b9c0-f7052f49a6f4.png)

**Comment by lx78WyY0J5** [comment]
You juste have to sponsor a public repo
https://docs.github.com/en/sponsors
https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account

**Reply by caioagiani** [reply]
Yes, you can sponsor any OpenSource User/Project, however the purpose of this article is to help people get their Badge for a minimal cost.

**Reply by lx78WyY0J5** [reply]
😆 I hasn't saw that's an "up" channel to get this badge
god bless my stupidity 🤣

**Comment by mnCath** [comment]
Up!

**Reply by caioagiani** [reply]
Tks

**Comment by lx78WyY0J5** [comment]
up

**Reply by caioagiani** [reply]
Thanks!!

**Comment by RianPacheco** [comment]
UP

**Reply by caioagiani** [reply]
Thanks!

**Comment by thallesbrandao** [comment]
Up!!! @caioagiani

**Reply by caioagiani** [reply]
Thanks @thallesbrandao

**Comment by jricciardi** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
